### PR TITLE
added owo language v2

### PR DESCRIPTION
**Pull request by Szoniko**
I'm sorry, please don't ban me

### CODE REVIEW

**Comment by nadirabbas** [comment]
please add the selector for the language too in `LanguageSelector.ts`

**Comment by Szoniko** [comment]
> please add the selector for the language too in `LanguageSelector.ts`

Done, I hope

File: kofta/src/app/components/LanguageSelector.tsx [hunk 1]
```diff
@@ -45,6 +45,7 @@ export const LanguageSelector: React.FC<LanguageSelectorProps> = ({
     { value: "tr", label: "🇹🇷 Turkish" },
     { value: "uk", label: "🇺🇦 Ukrainian" },
     { value: "ur", label: "🇵🇰 Urdu" },
+	  { value: "owo" label: "OwO english"},
```
Comment (tejxv) [inline kofta/src/app/components/LanguageSelector.tsx:48]: you missed a comma after `"owo"`

Comment (Szoniko) [reply kofta/src/app/components/LanguageSelector.tsx:48]: I'll fix that in a second, thanks!

**Pull request merged**